packages/vitest-pool-workers/src/pool/loopback.ts

```diff
@@ -326,11 +326,9 @@ function checkAllStorageOperationsResolved(
 			"",
 			separator,
 			`Failed to ${action} isolated storage stack frame in ${source}.`,
-			"This usually means your Worker tried to access storage outside of a test, or failed to cleanup storage passed across an RPC boundary.",
-			`In particular, we were unable to ${action} ${LIST_FORMAT.format(
-				failedProducts
-			)} storage.`,
-			"Ensure you `await` all `Promise`s that read or write to these services, and make sure you use the `using` keyword when passing data across JSRPC. See https://developers.cloudflare.com/workers/runtime-apis/rpc/lifecycle#explicit-resource-management for more details.",
+			`In particular, we were unable to ${action} ${LIST_FORMAT.format(failedProducts)} storage.`,
+			"This usually means your Worker access storage outside of a test, or some resources have not been disposed of properly.",
+			"See https://developers.cloudflare.com/workers/testing/vitest-integration/known-issues/#isolated-storage for common causes and suggested solution.",
```

Comment:
Why are we removing the actionable info from the error message? Could it be:

> Ensure you `await` all `Promise`s that read or write to these services, and make sure you use the `using` keyword when passing data across JSRPC. Refer to https://developers.cloudflare.com/workers/testing/vitest-integration/known-issues/#isolated-storage for more details.

Reply:
Right. Probably better to keep the actionable info there. Updated in https://github.com/cloudflare/workers-sdk/pull/8173/commits/541c1f774844cdbe214945597ca1406606ccfc96.